src/components/BusinessCaseReview/index.tsx

```diff
@@ -98,7 +98,7 @@ const BusinessCaseReview = ({
           </div>
         </div>
         {grtFeedbacks && grtFeedbacks.length > 0 && (
-          <div className="bg-gray-10 margin-top-3 padding-x-3 padding-top-3 padding-bottom-1">
+          <div className="bg-gray-10 margin-top-3 padding-x-3 padding-top-3 padding-bottom-1 margin-bottom-6">
```

Comment:
This change undoes work from https://github.com/CMSgov/easi-app/pull/1667; please manually reverse it.